py3status/modules/dpms.py

```diff
@@ -7,12 +7,16 @@
 by clicking on 'DPMS' in the status bar.
 
 Configuration parameters:
-    format_off: string to display when DPMS is disabled (default 'DPMS')
-    format_on: string to display when DPMS is enabled (default 'DPMS')
+    format: string to display (default '{query}')
+    format_off: string to display when disabled (default 'DPMS')
+    format_on: string to display when enabled (default 'DPMS')
+
+Format placeholders:
+    {query} display current dpms setting
```

Comment:
`{icon}` might be better see last comment on https://github.com/ultrabug/py3status/issues/617